src/Output.zig

```diff
@@ -69,6 +69,32 @@ pub const Layouts = enum {
     Full,
 };
 
+const LayoutName = struct {
+    name: []const u8,
+    layout: Layouts,
+};
+
+// zig fmt: off
+const LayoutNames = [_]LayoutName {
+    .{ .name = "TopMaster",    .layout = Layouts.TopMaster, },
+    .{ .name = "RightMaster",  .layout = Layouts.RightMaster, },
+    .{ .name = "BottomMaster", .layout = Layouts.BottomMaster, },
+    .{ .name = "LeftMaster",   .layout = Layouts.LeftMaster, },
+    .{ .name = "Full",         .layout = Layouts.Full, },
+};
+// zig fmt: on
+
+pub fn getLayoutByName(self: Self, name: []const u8) Layouts {
+    const current = for (LayoutNames) |current| {
```

Comment:
This variable is unused 
```suggestion
    for (LayoutNames) |current| {
```